### DISCUSSION TITLE
Abilty to generate data by census tract...

### DISCUSSION THREAD

**Discussion by dpjohnsoIU** [synthetichealth/synthea#1468] [General]
Does anyone know if it is possible to generate synthea data by census tract or census block group? I have been trying to alter the custom_demographics.csv but I still run into issues. It appears I would need to change other files in the setup. Has anyone tried to do this?

**Comment by jawalonoski** [comment]
Data is generated by State, or State and "named place" which can be a town or city (broadly speaking). In some states it could be a broader area like a county, depending on the data.

Otherwise, it does not support generating by census tract, zip code, etc.

You could potentially edit the code to make it do that, but that capability does not exist.
